subsystem/feature_support_levels_test.go

```diff
@@ -115,13 +115,13 @@ var _ = Describe("Feature support levels API", func() {
 				Expect(err).NotTo(HaveOccurred())
 			})
 
-			It("Update umn true fail on 4.13 with s390x with infra-env", func() {
-				expectedError := "cannot use Cluster Managed Networking because it's not compatible with the s390x architecture on version 4.13"
-				cluster, err := registerNewCluster("4.13-multi", common.S390xCPUArchitecture, models.ClusterHighAvailabilityModeFull, swag.Bool(true))
+			It("Update umn true fail on 4.15 with s390x with infra-env", func() {
```

Comment:
Is UMN valid on any version with s390x? Should we just remove the version info here?